src/main/java/ShortList.java

```diff
@@ -1,7 +1,7 @@
 public class ShortList {
   public static void main (String[] args) {
     List<Short> s = new ArrayList<>();
```

Comment:
Good job! The static analysis tool Error Prone reported that the below error [1] used to be here, but you removed it. Check out http://errorprone.info for more information.


[1] RecommenderTest/src/main/java/LongSet.java:3: error: cannot find symbol
    Set<Short> s = new HashSet<>();
    ^
  symbol:   class Set
  location: class LongSet